### ISSUE TITLE
CPU usage is high when tests are run in IntelliJ

### ISSUE THREAD

**Issue by sguptatw** [getgauge/Intellij-Plugin#330]
**Expected behavior**
CPU usage must be in an acceptable limit

**Actual behavior**
Java process CPU usage is close to 100 when tests are run from the IDE

**Steps to replicate**
* Open IntelliJ IDE, run specs
* Notice the CPU usage

**Version**
```
Gauge version: 0.9.8.nightly-2018-04-26
Commit Hash: 5bd699e

Plugins
-------
csharp (0.10.3.nightly-2018-04-11)
html-report (4.0.4.nightly-2018-04-11)
java (0.6.7.nightly-2018-04-24)
js (2.2.0.nightly-2018-04-20)
python (0.3.0.nightly-2018-04-26)
ruby (0.5.0.nightly-2018-04-16)
screenshot (0.0.1)
xml-report (0.2.1.nightly-2018-04-05)
```

**Comment by Apoorva-GA** [comment]
Some of observations:
- CPU usage(~90%) is heigh even with the released version of gauge(0.9.7) and gauge-java(0.6.6)
- CPU usage(~90%) is heigh even when tests are run from command line as well.

**Comment by vmatyusGitHub** [comment]
I have met with the same problem. I mainly experience high CPU load during debugging. When the IDE stops at a breakpoint starts to consume CPU load steadily. When I stop the debugging, the CPU usage starts to drop and decrease back to normal operation.

I am using Gauge plugin 0.3.16 with Gauge 1.0.6
I am not experiencing this with VSCode plugin, when I dealing with JS or Python. 
```
Plugins
-------
html-report (4.0.8)
java (0.7.2)
js (2.3.5)
python (0.3.5)
screenshot (0.0.1)
xml-report (0.2.2)
```

**Comment by zabil** [comment]
This might be related to IntelliJ versions https://github.com/getgauge/Intellij-Plugin/issues/421
Hoping this is resolved if you upgrade to IntelliJ IDEA 2019.3 

If not, please feel free to re-open.
